src/assets/small-icons/index.js

```diff
@@ -113,3 +113,4 @@ export { default as warningIcon } from './ic_warning.svg';
 export { default as whatappIcon } from './ic_whatapp.svg';
 export { default as workIcon } from './ic_work.svg';
 export { default as xingIcon } from './ic_xing.svg';
+export { default as contractIcon } from './ic_contract.svg';
```

Comment:
One question, how are you updating the icons? Manually or using the automation script?

The icons are a recurring topic and I think they're still out of sync: https://github.com/homeday-de/homeday-blocks/pull/533

Reply:
I updated manually... Thanks for letting me know. I'll investigate this topic today!

Reply:
Hey @volcanioo, let me know if you have any problem with the icons.

Reply:
Hey @ilyasmez, I have a couple of more icons and already talked with @viniciuskneves to get more information about our asset-import script. These days, I mainly focus on OKR meetings and Slider's new feature. For this reason, I couldn't work on these topics. I will add this icon with Magda's new icons that are reported on DS Channel in the next week.

Reply:
Perfect! Take your time :) I was just wondering if you're blocked and need help.
I'll add the `on hold` label to this PR, so we don't get review reminders for it on Slack, you can remove the label when it's ready for review again.

Reply:
Ah! Thanks for the hint! - I'll ping you when I'm done with assets-importer to get your feedback! 
![thank-you](https://user-images.githubusercontent.com/14016028/104606305-e5baec00-567f-11eb-9ae6-4ab1ab8809ae.gif)

Reply:
This icon changes will be removed here. Because we already did this in https://github.com/homeday-de/homeday-blocks/pull/646